package.json

```diff
@@ -25,9 +25,6 @@
   "lint-staged": {
     "*": [
       "prettier --ignore-unknown --write"
-    ],
```

Comment:
I removed this pre-commit hook as it's really slow (as it requires to `build` everything). The CI runs a `lint` check anyway, so it's not mandatory